.github/workflows/working_draft.yml

```
@@ -1,10 +1,9 @@
 name: generate_draft_spec
 on:
   push:
-    branches:
-      - "*"
-      - "!main"
-      - "!candidate_release"
+    branches-ignore:
+      - main
+      - candidate_release
```

I know it works without the double quotes, but `main.yml` and `candidate_release.yml` use them, so it would be be good to stay consistent.
```suggestion
      - "main"
      - "candidate_release"
```